index.html

```diff
@@ -7,24 +7,66 @@
     <title>David Porter</title>
   </head>
   <body>
-<section class="pagehead">
+    <header>
 
+    <section class="pagehead">
     <div class="myface">
       <img src="images/DavidPorter.jpg" alt="Profile Picture" class="mypic">
       <h1>David Porter</h1>
     </div>
+  </section>
+  <div id="menu">
+  <!-- <span><a href="https://twitter.com/TheGrimHEEP"><img src="images/Twitter-logo.jpg" alt="Twitter Profile" class="twit-logo"></a></span>
+  <span><a href="https://www.linkedin.com/in/david-porter-9584129?trk=hp-identity-photo"><img src="images/LinkedIn-logo.png" alt="LinkedIn" class="link-logo"></a></span>
+  <span><a href="https://github.com/thegrimheep"><img src="images/GitHub-logo.png" alt="GitHub" class="gitHub-logo"></a></span>
```

Comment:
If you are happy with the new links, it's probably safe to remove these commented-out lines of code.